test/unit/com/unifina/datasource/DataSourceEventQueueSpec.groovy

```diff
@@ -147,4 +146,43 @@ class DataSourceEventQueueSpec extends Specification {
 		})
 	}
 
+	void "timer drift"() {
```

Comment:
Years from now, it might be hard to remember what is being tested. Maybe something like:

```suggestion
	void "every full second between subsequent event timestamps is being reported"() {
```